src/routes/docs/advanced/security/+page.markdoc

```diff
@@ -0,0 +1,83 @@
+---
+layout: article
+title: Security
+description: Learn how Appwrite keeps your project, users, and data secure through security measures and compliance.
+---
+
+Appwrite helps you build secure apps by applying various security and compliance measures. 
+Appwrite is compliant with [GDPR]() and actively working toward [SOC 2]() and [HIPPA]() compliance.
+
+Appwrite also provides employs [enhanced password protection and encryption](), [rate limits](), [robust permission systems](), 
+[cross-site encryption protection](), and [HTTPS/SSL]() to protect you and your users' data.
+
+# Compliance {% #compliance %}
+
+The safeguarding of your and your users' data is taken seriously at Appwrite.
+Appwrite works to achieve compliance with a variety of standards to protect sensitive data, as well as maintain trust and credibility.
+
+{% cards %}
+{% cards_item href="/docs/advanced/security/gdpr"  title="GDPR" %}
+Appwrite is GDPR compliant. Learn about our measures, privacy policy, and find our data processing agreement.
+{% /cards_item %}
+
+{% cards_item href="/docs/advanced/security/pci"  title="PCI" %}
+Appwrite uses Stripe to handle payment and payment information securely. Learn about Appwrite's PCI compliance.
+{% /cards_item %}
+
+{% cards_item href="/docs/advanced/security/soc2"  title="SOC 2" %}
```

Comment:
```suggestion
{% cards_item href="/docs/advanced/security/soc2"  title="SOC 2" %}
```

When should this be SOC 2 vs SOC-2?

Reply:
Always SOC 2

https://cloud.google.com/security/compliance/soc-2

I see SOC 2 everywhere when I search